Labs/Lab2EC2Docker.md

```diff
@@ -109,7 +109,7 @@ docker --version
 
 Create a directory called html
 
-Edit a file index.html and add the following content
+Edit a file index.html inside the html directoyr and add the following content
```

Comment:
There's a small typo in the word directory. Please fix this and will accept this PR.